nltk/test/unit/test_tokenize.py

```diff
@@ -470,3 +462,51 @@ def test_punkt_tokenize_no_custom_lang_vars(self):
         expected = ["উপরাষ্ট্রপতি শ্রী এম ভেঙ্কাইয়া নাইডু সোমবার আই আই টি দিল্লির হীরক জয়ন্তী উদযাপনের উদ্বোধন করেছেন। অনলাইনের মাধ্যমে এই অনুষ্ঠানে কেন্দ্রীয় মানব সম্পদ উন্নয়নমন্ত্রী শ্রী রমেশ পোখরিয়াল ‘নিশাঙ্ক’  উপস্থিত ছিলেন। এই উপলক্ষ্যে উপরাষ্ট্রপতি হীরকজয়ন্তীর লোগো এবং ২০৩০-এর জন্য প্রতিষ্ঠানের লক্ষ্য ও পরিকল্পনার নথি প্রকাশ করেছেন।"]
 
         assert obj.tokenize(sentences) == expected
+
+    def test_punkt_debug_decisions_dot(self):
+        # Test debug_decisions on a text with two sentences, split by a dot.
+        tokenizer = punkt.PunktSentenceTokenizer()
+        txt = "Subject: Some subject. Attachments: Some attachemnts"
```

Comment:
spelling of `attachemnts`

Reply:
I've resolved this. The typo was already present in the example I took it from, but I suppose that the issue (obviously) still persists with the non-typo version. So, it's probably preferable to fix the typo, rather than stay true to the original example.